vowpalwabbit/cb_explore_adf.cc

```diff
@@ -315,18 +299,21 @@ void finish(cb_explore_adf& data)
 //are specified. We print the first action and probability, based on
 //ordering by scores in the final output.
 
-void output_example(vw& all, cb_explore_adf& c, example& ec, v_array<example*>* ec_seq)
+void output_example(vw& all, cb_explore_adf& c, multi_ex& ec_seq)
 {
+  if (ec_seq.size() <= 0) return;
+
+  auto& ec = *ec_seq[0];
   if (CB_ALGS::example_is_newline_not_header(ec)) return;
```

Comment:
We should not need this line now, right?

Reply:
yep.  took it out